pkgs/applications/networking/cawbird/default.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv, fetchFromGitHub, glib, gtk3, json-glib, sqlite, libsoup, gettext, vala
+, meson, ninja, pkgconfig, gnome3, gst_all_1, wrapGAppsHook, gobject-introspection
+, glib-networking, python3 }:
+
+stdenv.mkDerivation rec {
+  version = "1.0.1";
+  pname = "cawbird";
+
+  src = fetchFromGitHub {
+    owner = "IBBoard";
+    repo = "cawbird";
+    rev = "v${version}";
+    sha256 = "sha256:0bk33fh32nnv6ya6j0ij34abw6a3g6m8fq13303slhhja8xhvmb1";
+  };
+
+  nativeBuildInputs = [
+    meson ninja vala pkgconfig wrapGAppsHook python3
+    gobject-introspection # for setup hook
+  ];
+
+  buildInputs = [
+    glib gtk3 json-glib sqlite libsoup gettext gnome3.dconf gnome3.gspell glib-networking
+  ] ++ (with gst_all_1; [ gstreamer gst-plugins-base gst-plugins-bad (gst-plugins-good.override { gtkSupport = true; }) gst-libav ]);
+
+  postPatch = ''
+    chmod +x data/meson_post_install.py # patchShebangs requires executable file
+    patchShebangs data/meson_post_install.py
+  '';
+
+  meta = {
+    description = "Native GTK Twitter client for the Linux desktop";
+    longDescription = "Cawbird is a modern, easy and fun Twitter client. Fork of the discontinued Corebird.";
+    homepage = https://corebird.baedert.org/;
+    license = stdenv.lib.licenses.gpl3;
+    platforms = stdenv.lib.platforms.linux;
+    maintainers = with stdenv.lib.maintainers; [ jonafato schmittlauch ];
```

Comment:
should jonafato be here?

Reply:
That's his choice. As I based this nix expression on the one for Corebird where he is the maintainer, I'd assume that he's also interested in maintaining the continuation fork.

On the other hand the project has been non-functional since the beginning of this year, so he might have lost interest.

@jonafato

Reply:
That's fair, not sure how often is package is likely to receive attention.

Reply:
I leave @jonafato as a maintainer until he complains or removes himself.